_data-prepper/peer-forwarder.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: default
+title: Peer Forwarder
+nav_order: 12
+---
+
+# Peer Forwarder
+
+Peer Forwarder is an HTTP service that performs peer forwarding of an `event` between Data Prepper nodes for aggregation. Currently, Peer Forwarder is supported by the `aggregate`, `service_map_stateful`, and `otel_trace_raw` processors.
+
+Peer Forwarder groups events based on the identification keys provided by the processors. For `service_map_stateful` and `otel_trace_raw`, the identification key is `traceId` by default and cannot be configured. The `aggregate` processor is configured using the `identification_keys` configuration option. From here, you can specify which keys to use for Peer Forwarder. See [Aggregate Processor page](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#identification_keys) for more information about identification keys.
+
+Peer discovery is currently provided by a static list, a DNS record lookup, or AWS Cloud Map.  
```

Comment:
@dlvenable Can we get some more information on peer discovery to better establish context here?

Reply:
The peers are other Data Prepper nodes in the same cluster. So "peer discovery" is where Data Prepper finds the other nodes that it will communicate with.